CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## In Development
+
+* Switch E2E testing to [Ansible Molecule](https://molecule.readthedocs.io/en/latest/). (by @mamercad)
```

Comment:
For simplicity/maintenance, we don't maintain the Changelog for this repo, but generate it for releases here:
https://github.com/StackStorm/ansible-st2/releases

So worth removing the file.

```suggestion
```